Sources/ApexyAlamofire/AlamofireClient.swift

```diff
@@ -5,6 +5,7 @@
 //  Copyright © 2019 RedMadRobot. All rights reserved.
 //
 
+import Apexy
```

Comment:
Is it necessary?

Reply:
You can check - there is `Cannot find type '' in scope` error everywhere, if `import Apexy` is not added